ansible/roles/service/hippo_authoring/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+hippo_authoring:
+  state: present
+  public_fqdn: "{{ public_fqdn }}"
+
+  aws:
+    instance_count: 1
+    instance_monitoring: "false"
```

Comment:
Why disabled?

Reply:
by default we don't want to generate any extra costs for non-production environments